server/routes/taskList.js

```diff
@@ -106,3 +111,11 @@ module.exports = ({ prisonerService, licenceService, audit }) => router => {
 
   return router
 }
+
+const getAllErrorMessages = errorCodes => {
+  const errorMsgs = Object.keys(errorCodes)
+  return {
+    errMsg1: taskListErrors[errorMsgs[0]],
+    errMsg2: taskListErrors[errorMsgs[1]],
```

Comment:
Think this would be nicer to represent as a list rather than an object - in the template you are only using list operators: isEmpty, iterate all values and it would be simpler to represent here too: 
`return errorCodes.map(error => taskListErrors[error])`

Reply:
ok